rotkehlchen/api/v1/schemas.py

```diff
@@ -945,12 +946,37 @@ def make_history_base_entry(
         ) -> dict[str, Any]:
             return {'event': EthWithdrawalEvent(**data)}
 
+    class CreateAssetMovementEventSchema(BaseSchema):
+        event_type = SerializableEnumField(
+            enum_class=HistoryEventType,
+            exclude_types=[
+                event_type for event_type in HistoryEventType
+                if event_type not in (HistoryEventType.DEPOSIT, HistoryEventType.WITHDRAWAL)
+            ],
+            required=True,
+        )
+        asset = AssetField(required=True, expected_type=Asset, form_with_incomplete_data=True)
+        event_identifier = fields.String(required=False, load_default=None)
+        unique_id = fields.String(required=False, load_default=None)
+        extra_data = fields.Dict(load_default=None)
```

Comment:
Since this is something coming from the API, thus the user we don't need to pass the extra data directly. Just have fields for each possible value and type check each field on its own merit. We can't have any kind of extra data there, can we?